databank/models.py

```diff
@@ -535,6 +535,11 @@ class CountryOverview(models.Model):
         verbose_name=_('HDR GII'),
         null=True, blank=True
     )
+    fdrs_data_fetched_year = models.CharField(
+        verbose_name=_('FDRS Data Fetched Year'),
+        null=True, blank=True,
+        max_length=50
+    )
```

Comment:
Any reason why we are storing year in string?

Reply:
We get year as string from data source and populating that here.